rs/bitcoin/ckbtc/minter/src/updates/retrieve_btc.rs

```diff
@@ -286,17 +286,20 @@ pub async fn retrieve_btc_with_approval(
         ));
     }
 
-    let kyt_principal = read_state(|s| {
-        s.kyt_principal
-            .expect("BUG: upgrade procedure must ensure that the KYT principal is set")
+    let btc_checker_principal = read_state(|s| {
+        s.btc_checker_principal
+            .expect("BUG: upgrade procedure must ensure that the bitcoin checker principal is set")
             .get()
             .into()
     });
 
-    match kyt_check_address(kyt_principal, parsed_address.display(btc_network)).await {
+    match check_address(btc_checker_principal, parsed_address.display(btc_network)).await {
         Err(error) => {
             return Err(RetrieveBtcWithApprovalError::GenericError {
-                error_message: format!("Failed to call KYT canister with error: {:?}", error),
+                error_message: format!(
+                    "Failed to call bitcoin checker canister with error: {:?}",
+                    error
+                ),
                 error_code: ErrorCode::KytCallFailed as u64,
```

Comment:
Should this variant `ErrorCode::KytCallFailed` be renamed as well?

Reply:
Good catch! Now renamed